lib/Cache/RedisDB.pm

```diff
@@ -133,6 +133,34 @@ sub del {
     return redis->del(map { "${namespace}::$_" } @keys);
 }
 
+=head2 keys($namespace)
+
+Return a list of all known keys in the provided I<$namespace>.
+
+=cut
+
+sub keys {
+    my ($self, $namespace) = @_;
+    my $prefix = $namespace . '::';
```

Comment:
Hrm.  Let me see how the rest of this module handles this.  It's a good point, but I wanna make sure the response is consistent.